### DISCUSSION TITLE
Confusion regarding supported versions for Gloo Edge

### DISCUSSION THREAD

**Discussion by shazib-summar** [kgateway-dev/kgateway#8671] [Q&A]
This page at https://docs.solo.io/gloo-edge/latest/reference/support/ says at the top that

<img width="1042" alt="image" src="https://github.com/solo-io/gloo/assets/27294025/e223fecd-6584-413f-9d0e-234fade7f8d6">

> Review the following information about supported release versions for Gloo Edge Enterprise and Gloo Edge OSS (open source).

indicating that the information on the page is for both Gloo Edge Enterprise and for Gloo Edge OSS. But the [Supported versions](https://docs.solo.io/gloo-edge/latest/reference/support/#supported-versions) section is confusing as it only mentions Gloo Edge Enterprise

<img width="1121" alt="image" src="https://github.com/solo-io/gloo/assets/27294025/63244ed3-fa01-4b1a-98e2-208dcaaa559f">

Where can I find/figure out the supported versions for Gloo Edge OSS. Kindly help.

**Comment by craigbox** [comment]
Hi @shazib-summar,

This question is outdated due to "Gloo Edge OSS" becoming kgateway.  As a fully community-led project, there is no paid 'support'; you can get that from a project that is based on kgateway, such as Gloo Gateway.  The page you linked refers to Solo's support for the Gloo product. 

Hopefully that helps - if not, feel free to [drop by the Solo Slack](https://solo-io.slack.com/join/shared_invite/zt-79jfjpnp-bfcfKuN88xMywWd2xQpTOA)!
